commerce-cloud-code/app_yourshophere/cartridge/partials/header/searchbox.js

```diff
@@ -9,6 +9,7 @@ function createModel(params) {
     const model = {
         placeholderText: params.placeholderText || 'Search',
         accesssibilityLabel: params.accesssibilityLabel || 'Enter keyword or Item No',
+        checked: request.httpParameterMap.q.submitted,
```

Comment:
what is that doing?

Reply:
It no longer does anything, not did it ever as there is no querystring and no parameters.